### ISSUE TITLE
compare function single copy ortholog discrepancy

### ISSUE THREAD

**Issue by dns265** [nextgenusfs/funannotate#246]
I'm using version 1.5.0 but am currently unable to upgrade to 1.5.1 on shared environment.

I ran funannotate compare on 5 annotations generated using funannotate. While the stats output describes 5,639 single copy orthologs, no phylogeny was generated because "there seem to be 0 single copy orthologs" in proteinortho output
[funannotate-compare.log](https://github.com/nextgenusfs/funannotate/files/2687237/funannotate-compare.log)

**Comment by dns265** [comment]
It ran successfully when the outgroup was omitted.  I was primarily testing the function using a subset of my data, so before I run the full analysis I'll be sure to add a custom outgroup using the same BUSCO database.  We're in the process of upgrading to funannotate 1.5.1 to try and make sure the iprscan annotations are actually included in the gbk files.  Once that is done I'll retry the compare function.  I'll submit a new issue if using a consistent BUSCO database doesn't fix things.  Thanks!
